WolvenKit.App/Helpers/ProjectResourceTools.cs

```diff
@@ -485,6 +486,49 @@ public void ReplacePathInProject(Cp77Project? activeProject, ResourcePath oldPat
             }
         });
 
+
+        if (oldPath.GetResolvedText() is string oldPathStr && !string.IsNullOrEmpty(oldPathStr) &&
+            newPath.GetResolvedText() is string newPathStr && !string.IsNullOrEmpty(newPathStr))
+        {
+            var oldPathWithForwardSlashes = oldPathStr.Replace("\\", "/");
+            var newPathWithForwardSlashes = newPathStr.Replace("\\", "/");
+            
+            var resourceFiles =
+                Directory.GetFiles(activeProject.ResourcesDirectory, "*.*", SearchOption.AllDirectories);
```

Comment:
This should only get specific text files in resources with known extensions imo: .lua, .script, .tweak, .yaml, .reds and .json

As it is now, it would also replace stuff in non-text files